service-front/module/Application/src/Controller/DocumentCheckController.php

```diff
@@ -43,6 +43,14 @@ public function nationalInsuranceNumberAction(): ViewModel
         $form = $this->createForm(NationalInsuranceNumber::class);
         $detailsData = $this->opgApiService->getDetailsData($uuid);
 
+        /**
+         * @psalm-suppress InvalidArrayOffset
+         */
```

Comment:
To avoid needing this suppressions, add `identityCheckPassed` to [the shape definition for CaseData](https://github.com/ministryofjustice/opg-paper-identity/blob/main/service-front/module/Application/src/Contracts/OpgApiServiceInterface.php#L24-L64)